Resources/views/Admin/Security/Resetting/checkEmail.html.twig

```diff
@@ -0,0 +1,30 @@
+{% extends base_template %}
+
+{% block sonata_nav %}
+{% endblock sonata_nav %}
```

Comment:
if this block is empty i would recommend to write this in one line, but i'm not sure if the other sonata templates have a line break

ping @rande